libs/ledger-live-common/src/families/vechain/specs.ts

```diff
@@ -0,0 +1,165 @@
+        account,
+        siblings,
+        bridge,
+      }: TransactionArg<Transaction>): TransactionRes<Transaction> => {
+        const sibling = pickSiblings(siblings, 2);
+        const recipient = sibling.freshAddress;
+        if (!account.subAccounts || !account.subAccounts[0]) throw new Error("no VTHO account");
+        const tokenAccount = account.subAccounts[0];
+        const transaction = bridge.createTransaction(tokenAccount);
+        transaction.subAccountId = account.subAccounts[0].id;
+        const amount = tokenAccount.balance.div(2).integerValue();
+        const updates = [{ amount }, { recipient }, { subAccountId: account.subAccounts[0].id }];
+        return {
+          transaction,
+          updates,
+        };
+      },
+      test: ({
+        account,
+        optimisticOperation,
+      }: TransactionTestInput<Transaction>): void | undefined => {
+        botTest("checks the existence of the operation in the subaccount", () => {
+          assert(account?.subAccounts?.[0]);
+          expect(
+            account.subAccounts[0].operations.find(
+              op => op.id === optimisticOperation?.subOperations?.[0].id,
+            ),
+          ).toBeTruthy();
+        });
+      },
+    },
+    {
+      name: "move all VET",
+      maxRun: 1,
+      transaction: ({
+        account,
+        siblings,
+        bridge,
+        maxSpendable,
+      }: TransactionArg<Transaction>): TransactionRes<Transaction> => {
+        const sibling = pickSiblings(siblings, 4);
+        const recipient = sibling.freshAddress;
+        const transaction = bridge.createTransaction(account);
+        transaction.useAllAmount = true;
+        const amount = maxSpendable.integerValue();
+        const updates = [{ amount }, { recipient }];
+        return {
+          transaction,
+          updates,
+        };
+      },
+      test: ({ account }: TransactionTestInput<Transaction>): void | undefined => {
+        botTest("account balance decreased with operation value", () =>
+          expect(account.balance.toString()).toBe(new BigNumber(0).toString()),
+        );
+      },
+    },
+    {
+      name: "move all VTHO",
+      maxRun: 1,
+      transaction: ({
+        account,
+        siblings,
+        bridge,
+      }: TransactionArg<Transaction>): TransactionRes<Transaction> => {
+        const sibling = pickSiblings(siblings, 4);
+        const recipient = sibling.freshAddress;
+        if (
+          !account.subAccounts ||
+          !account.subAccounts[0] ||
+          !(account.subAccounts[0].type == "TokenAccount")
+        )
+          throw new Error("no VTHO account");
+        const transaction = bridge.createTransaction(account);
+        transaction.useAllAmount = true;
+        transaction.subAccountId = account.subAccounts[0].id;
+        const amount = account.subAccounts[0].spendableBalance.integerValue();
+        const updates = [{ amount }, { recipient }, { subAccountId: account.subAccounts[0].id }];
```

Comment:
In theory you don't need to provide `amount` at all, when `useAllAmount` is used

Reply:
I'll remove it for now